app/views/devise/passwords/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Change your password</h2>
```

Comment:
If we didn't make any change in the devise view we can remove them.
But its upto you how you want to manage this

Reply:
sure